### PR TITLE
Remove z-index from handsontableInputHolder and add it to handsontableEditor 

### PR DESCRIPTION
**Pull request by pnowak**
### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. #5105 

### Checklist:

- [x] My code follows the code style of this project,
- [ ] My change requires a change to the documentation.

### CODE REVIEW

File: test/e2e/editors/passwordEditor.spec.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ describe('PasswordEditor', () => {
     const editorHolder = $('.handsontableInputHolder');
     const editor = editorHolder.find('.handsontableInput');
 
-    expect(parseInt(editorHolder.css('z-index'), 10)).toBeGreaterThan(0);
+    expect(editorHolder.is(':visible')).toBe(true);
```
Comment (swistach) [inline test/e2e/editors/passwordEditor.spec.js]: Are you sure? We've changed it during an introduction of the IME support.

**Comment by wojciechczerniak** [comment]
@pnowak @swistach Haven't we refactored this recently? Can we close this outdated PR?

**Comment by pnowak** [comment]
Yes, we can close this PR.

**Comment by pnowak** [comment]
In this PR https://github.com/handsontable/handsontable/pull/6269 we took care of `z-index` more holistically.

**Pull request closed without merging**